tests/integration_tests/instances.py

```diff
@@ -13,7 +13,9 @@
 try:
     from typing import TYPE_CHECKING
     if TYPE_CHECKING:
-        from tests.integration_tests.clouds import IntegrationCloud
+        from tests.integration_tests.clouds import (  # noqa: F401
```

Comment:
Why make this 3 lines?

Reply:
Adding `# noqa: F401` at the end of line crosses 80 characters, so broke this into multiple lines.